src/aleph/vm/orchestrator/messages.py

```diff
@@ -69,3 +71,15 @@ async def load_updated_message(
     message = copy.deepcopy(original_message)
     await update_message(message)
     return message, original_message
+
+
+async def get_message_status(item_hash: ItemHash) -> MessageStatus:
+    """Fetch the status of an execution from the reference API server."""
```

Comment:
The rest of the code fetch messages from connector but here it is using the api server. I'm supposing there is a reason for that but it would be nice to have it explained and written.  Especially for maintainability

Reply:
Indeed, so I added the explanation on the method description :smiley: